python/pip_install/parse_requirements_to_bzl/__init__.py

```diff
@@ -209,9 +209,11 @@ def main() -> None:
     arguments.parse_common_args(parser)
     args = parser.parse_args()
 
+    whl_library_args = parse_whl_library_args(args)
+
     # Check for any annotations which match packages in the locked requirements file
     install_requirements = parse_install_requirements(
-        args.requirements_lock, args.extra_pip_args
+        args.requirements_lock, whl_library_args["extra_pip_args"]
```

Comment:
This matches the expectations before:
https://github.com/bazelbuild/rules_python/blob/0.6.0/python/pip_install/parse_requirements_to_bzl/__init__.py#L68-L76